src/docfx/build/Build.cs

```diff
@@ -106,7 +100,7 @@ async Task BuildOneFile(Document file, Action<Document> buildChild)
                 bool ShouldBuildFile(Document file)
                 {
                     // source content in a localization docset
-                    if (docset.FallbackDocset != null && file.Docset.LocalizationDocset != null)
+                    if (docset.IsLocalized() && !file.Docset.IsLocalized())
```

Comment:
>IsLocalized [](start = 31, length = 11)

What's the difference between FallbackDocset and LocalizationDocset? Why change from check FallbackDocset to check LocalizationDocset here?

Reply:
it's used for different purpose, here is only a flag to see if the current docset is localized or not